openslides_backend/action/generics.py

```diff
@@ -336,7 +337,7 @@ def delete_action_prepare_dataset(self, payload: ActionPayload) -> DataSet:
             relation_fields: List[Tuple[str, BaseRelationField]] = []
             # Gather all delete actions with payload and also all models to be deleted
             delete_actions: List[Tuple[Type[Action], ActionPayload]] = []
-            additional_relation_models: ModelMap = {}
+            additional_relation_models: ModelMap = {this_fqid: DeletedModel()}
```

Comment:
This was missing and the reason why the test failed

Reply:
Ok, nice.